src/mango/test/13-stable-update-test.py

```diff
@@ -44,6 +45,7 @@ def setUp(self):
         self.db.create_index(["name"], ddoc="foo")
         self.db.save_docs(copy.deepcopy(DOCS1))
 
+    @unittest.skip("this FDB doesn't support this")
     def test_update_updates_view_when_specified(self):
```

Comment:
Don't skip tests. Either change the test to assert the new behavior or remove it altogether if its just doesn't make sense. Though it sure seems to me like update=false isn't something that's completely not applicable even with the update-in-transaction semantics due to the initial build being a thing.